anime_downloader/sites/init.py

```diff
@@ -2,6 +2,7 @@
 
 ALL_ANIME_SITES = [
     # ('filename', 'sitename', 'classname')
+    ('animeonline360','animeonline360','AnimeOnline'),
```

Comment:
I'd personally prefer if you removed the 360, it's too long

Reply:
from the filename? ok

Reply:
sitename, to make `--provider` shorter